student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom, :parent_permission
```

Comment:
- [ ] Please 🙏🏽 remove this line, it is not necessary to use attr_reader for any variable in this class, you can see this article of [how to use...](https://www.rubyguides.com/2018/11/attr_accessor/)